actors/builtin/miner/policy.go

```diff
@@ -71,34 +76,36 @@ var MaxSealDuration = map[abi.RegisteredSealProof]abi.ChainEpoch{
+// Filecoin Parameter: Lookback from the deadline's challenge window opening from which to sample chain randomness for the WindowPoSt challenge seed.
 // This lookback exists so that deadline windows can be non-overlapping (which make the programming simpler)
-// but without making the miner wait for chain stability before being able to start on PoSt computation.
-// The challenge is available this many epochs before the window is actually open to receiving a PoSt.
+// Motivation: A miner can start the WindowPoSt computation without waiting for chain stability. This value cannot be too large since it could compromise the rationality of honest storage (due to WindowPoSt cost assumptions)
 const WPoStChallengeLookback = abi.ChainEpoch(20)
 
-// Minimum period before a deadline's challenge window opens that a fault must be declared for that deadline.
-// This lookback must not be less than WPoStChallengeLookback lest a malicious miner be able to selectively declare
-// faults after learning the challenge value.
+// Filecoin Parameter: Minimum period between fault declaration and the next deadline opening.
+// If the number of epochs between fault declaration and deadline's challenge window opening is lower than FaultDeclarationCutoff,
+// the fault declaration is considered invalid for that deadline.
+// Motivation: This guarantees that a miner is not likely to successfully fork the chain and declare a fault after seeing the challenges.
 const FaultDeclarationCutoff = WPoStChallengeLookback + 50
 
-// The maximum age of a fault before the sector is terminated.
+// Filecoin Parameter: The maximum age of a fault before the sector is terminated.
+// Motivation: This guarantees to clients that a Filecoin miner cannot lose the file for longer than 14 days.
 const FaultMaxAge = WPoStProvingPeriod*14
 
-// Staging period for a miner worker key change.
-// Finality is a harsh delay for a miner who has lost their worker key, as the miner will miss Window PoSts until
+// Filecoin Parameter: Staging period for a miner worker key change.
+// Future improvement: Finality is a harsh delay for a miner who has lost their worker key, as the miner will miss Window PoSts until
 // it can be changed. It's the only safe value, though. We may implement a mitigation mechanism such as a second
 // key or allowing the owner account to submit PoSts while a key change is pending.
+// Motivation: This guarantees that a miner cannot choose a more favorable worker key that wins leader elections.
 const WorkerKeyChangeDelay = ChainFinality
 
-// Maximum number of epochs past the current epoch a sector may be set to expire.
+// Filecoin Parameter: Maximum number of epochs past the current epoch a sector may be set to expire.
 // The actual maximum extension will be the minimum of CurrEpoch + MaximumSectorExpirationExtension
 // and sector.ActivationEpoch+sealProof.SectorMaximumLifetime()
 const MaxSectorExpirationExtension = builtin.EpochsInYear
```

Comment:
**Clarification requested:** Why is this set to one year? What is the motivation? It seems that the MaxSectorLifetime is respected in the code, can we have confirmation?

Reply:
No idea where it comes from.  I believe it is respected in the code when processing sector deadlines and expirations.  cc @acruikshank who knows this area better than me

Reply:
This was introduced when max sector lifetime determined by PoRep is 3-5 times greater than the max sector lifetime that we are allowed to add at a time (1-2 year range). We can either remove it now but we may add this back later or we just remove it entirely and consider bringing it back when we switch to a different PoRep.

Reply:
Will fix: This is how much a miner is allowed to extend every time they try to extend.